lib/index.js

```diff
@@ -0,0 +1,38 @@
+const Request = require('./request')
+const root = 'https://track.customer.io/api/v1'
+
+module.exports = class CustomerIO {
+  constructor(siteid, apikey) {
+    this.siteid = siteid
+    this.apikey = apikey
+    this.request = new Request(this.siteid, this.apikey)
+  }
+
+  identify(id, data = {}) {
+    return this.request.put(`${root}/customers/${id}`, data)
+  }
+
+  destroy(id) {
+    return this.request.destroy(`${root}/customers/${id}`)
+  }
+
+  track(id, data = {}) {
+    return typeof id === 'object'
+      ? this.request.post(`${root}/events`, id)
+      : this.request.post(`${root}/customers/${id}/events`, data)
+  }
+
+  trackPageView(id, path) {
+    return this.request.post(`${root}/customers/${id}/events`, {
+      type: 'page',
+      name: path
+    })
+  }
+
+  triggerBroadcast(id, data, recipients) {
+    return this.request.post(`${root}/campaigns/${id}/triggers`, {
```

Comment:
This won't actually work for triggered broadcasts because they use a different API endpoint. `https://api.customer.io/v1/api/campaigns/${id}/triggers` instead of `https://track.customer.io/api/v1/campaigns/${id}/triggers`

For now only track.customer.io API methods are in the libraries because we're still determining the best path forward to supporting the two distinct APIs we provide.

Reply:
Good catch! I'll update this now, and will also separate the track method in the way you described. I considered this initially, but decided to go the way I did because it's the same endpoint, and it was a little smoother of an interface. But the rationale you have there makes perfect sense and I'll change it over.